Makefile

```diff
@@ -0,0 +1,58 @@
+#-- LICENSE: The MIT License(MIT) 
+# Copyright(c) 2021 Dr Ashton Fagg <ashton@fagg.id.au>
+#
```

Comment:
What is the point of a license notice in a makefile, really?

Reply:
I'll remove it. I've seen this before in other projects, and my editor happens to do it automatically.